llvm/lib/Target/RISCV/RISCVFeatures.td

```diff
@@ -996,6 +996,12 @@ def TuneShortForwardBranchOpt
 def HasShortForwardBranchOpt : Predicate<"Subtarget->hasShortForwardBranchOpt()">;
 def NoShortForwardBranchOpt : Predicate<"!Subtarget->hasShortForwardBranchOpt()">;
 
+def TuneCMOVBranchOpt
```

Comment:
This naming is hard for me to parse at a glance.  A couple ideas:
* TuneCMovFusion
* TuneShortForwardBranchCMovOnly